reporting_guidelines.md

```diff
@@ -0,0 +1,51 @@
+
+In your report please include:
+
+* Your contact info for follow-up contact.
+* Names (legal, nicknames, or pseudonyms) of any individuals involved.
+* If there were other witnesses besides you, please try to include them as well.
+* When and where the incident occurred. Please be as specific as possible.
+* Your account of what occurred. 
+* If there is a publicly available record (e.g. a mailing list archive or a public IRC logger) please include a link.
+* Any extra context you believe existed for the incident.
+* If you believe this incident is ongoing.
+* Any other information you believe we should have.
+
+## What happens after you file a report?
+
+You will receive an email from the Anitab.org Open Source's Code of Conduct response team acknowledging receipt as soon as possible, but within 48 hours.
+
+The working group will immediately meet to review the incident and determine:
+
+* What happened.
+* Whether this event constitutes a code of conduct violation.
+* What kind of response is appropriate.
+
+If this is determined to be an ongoing incident or a threat to physical safety, the team's immediate priority will be to protect everyone involved. This means we may delay an "official" response until we believe that the situation has ended and that everyone is physically safe.
+
+Once the team has a complete account of the events they will make a decision as to how to respond. Responses may include:
+
+Nothing (if we determine no code of conduct violation occurred).
+* A private reprimand from the Code of Conduct response team to the individual(s) involved.
+* A public reprimand.
+* An imposed vacation (i.e. asking someone to "take a week off" from a mailing list or IRC).
+* A permanent or temporary ban from some or all of Anitab.org spaces (events, meetings, mailing lists, IRC, etc.)
+* A request to engage in mediation and/or an accountability plan.We'll respond within one week to the person who filed the report with either a resolution or an explanation of why the situation is not yet resolved.
+
+Once we've determined our final action, we'll contact the original reporter to let them know what action (if any) we'll be taking. We'll take into account feedback from the reporter on the appropriateness of our response, but our response will be determined by what will be best for community safety.
+
+Finally, the response team will make a report on the situation to the Anitab.org's Open Source Board. The board may choose to issue a public report of the incident or take additional actions.
+
+## Appealing the response
+
+Only permanent resolutions (such as bans) may be appealed. To appeal a decision of the working group, contact the community admins at osdghc@anitab.org with your appeal and we will review the case.
+
+_Revision 1.0, DRAFT, proposed June 2014_
```

Comment:
Should we replace this with today and point to the mentoring source or should we leave this as is?

Reply:
Modified to May 2021 with reference to the [AnitaB mentorship reporting guidelines](https://github.com/anitab-org/mentorship-backend/blob/develop/docs/reporting_guidelines.md). Let me know what you think!